monitoring/grafana/grr_grafanalib_dashboards/config.py

```diff
@@ -0,0 +1,5 @@
+from grr_grafanalib_dashboards import reusable_panels
+
+DATA_SOURCE = "grr-server"
+ACTIVE_PROCESSES_CONDITION = 1
```

Comment:
Please add a comment documenting what this value means. Same for "DATA_SOURCE".

Reply:
If it's ok, I think comments will clutter it a bit; I'll try to make the variables' names more descriptive, as they are pretty straightforward. Let me know what you think.

Reply:
ACTIVE_PROCESSES_ALERTING_CONDITION is a better name but it's still confusing. Is this a number of processes? Will it alert if there's one process or less than one process? Comments would be helpful.
Also for DATA_SOURCE, having a comment would help - where does the value is coming from, am I supposed to change it? Understanding this from the comment would be great.

In general, I'd say that if comments clutter the code, that's great - because most of the code is not cluttered by comments and is sometimes hard to read precisely because of that :)

Reply:
You're right; I elaborated in the comments, and also added a relevant line in the monitoring documentation, in [this commit](https://github.com/google/grr-doc/pull/117/commits/b1507d3a95a8b405c3e3db95888e71d9c2eb210b), as the definition of `DATA_SOURCE` depends on how exactly did the user set up Grafana during installation.